src/app/item-feed/ItemFeedSidebar.m.scss

```diff
@@ -38,7 +38,7 @@
   right: 100%;
   color: var(--theme-text);
   padding: 8px;
-  top: -25px;
+  top: 25px;
```

Comment:
This was/is intentional to prevent the item feed button from blending in with the header in the dark theme 

<img width="129" alt="image" src="https://github.com/DestinyItemManager/DIM/assets/1396158/deb05718-a1b3-4797-8366-8fb0c72a196d">

While not strictly necessary, breaking it out of the header improves it's visibility with the dark theme and i don't think it adversely affects how it is accessed in the regular theme?

Reply:
Makes sense, thanks 👍